### DISCUSSION TITLE
Need more details and clarification

### DISCUSSION THREAD

**Discussion by tobsowo** [linagora/tmail-flutter#2282] [General]
I get to know about Tmail through Stalwart and I have a couple of questions:
- Is it possible to login to multiple accounts on same device?
- Is there support for Application Password so I can integrate a mailbox with a third party app?
- is there an Admin Manager to mange multiple user accounts?
- BIMI support for brand logos?

**Comment by hoangdat** [comment]
> Is it possible to login to multiple accounts on same device?
> is there an Admin Manager to mange multiple user accounts?

Multiple account is on our roadmap, but still not priority in short and middle term

> Is there support for Application Password so I can integrate a mailbox with a third party app?

Srr, I can not get your point. Can you more details?

> BIMI support for brand logos?

Interesting things, we will consider adding it to roadmap

**Reply by tobsowo** [reply]
Application are secondary password to authenticate with third party app that requires SMTP integration.

See this links

Start from 35 seconds
https://m.youtube.com/watch?v=Pld5m9UJ8SE&embeds_referring_euri=https%3A%2F%2Fwebmail.wildduck.email%2F&source_ve_path=OTY3MTQ&feature=emb_imp_woyt

https://support.google.com/accounts/answer/185833?hl=en

**Reply by hoangdat** [reply]
@chibenwa di you have any idea about it?

**Comment by tobsowo** [comment]
I'm using tmail with Stalwart but it seems sending email after it is connected is not working for the JMAP standard for email submission.

Please can you helps work on this and should I create an issue?

**Reply by hoangdat** [reply]
This was also discuss here https://github.com/linagora/tmail-flutter/discussions/2230. Hope we can do it soon.

**Comment by hoangdat** [comment]
Hi @tobsowo nice to say that we are verifying TeamMail with stalwart backend. The work is in progress: https://github.com/linagora/tmail-flutter/pull/2315

**Reply by tobsowo** [reply]
Thank you for the update
